indexing-service/src/main/java/org/apache/druid/indexing/overlord/duty/TaskLogAutoCleanerConfig.java

```diff
@@ -43,18 +43,18 @@ public class TaskLogAutoCleanerConfig
   private final long delay;
 
   @JsonProperty
-  private final long durationToRetain;
+  private final long durationToRetainMs;
 
   @JsonCreator
   public TaskLogAutoCleanerConfig(
       @JsonProperty("enabled") boolean enabled,
       @JsonProperty("initialDelay") Long initialDelay,
       @JsonProperty("delay") Long delay,
-      @JsonProperty("durationToRetain") Long durationToRetain
+      @JsonProperty("durationToRetain") Long durationToRetainMs
```

Comment:
If this is being changed, other fields such as `initialDelay`, `delay` should be changed too.
I think it is okay to keep it the way it is as the name `durationToRetain` is being used in several other configs too. You could add javadocs to clarify the usage though.

Reply:
Reverted and added the javadoc instead.